integration-cli/docker_cli_run_test.go

```diff
@@ -61,6 +61,15 @@ func (s *DockerSuite) TestRunWithoutMemoryswapLimit(c *check.C) {
 	}
 }
 
+func (s *DockerSuite) TestRunWithSwappiness(c *check.C) {
+	testRequires(c, NativeExecDriver)
```

Comment:
@cpuguy83 , ofcourse corresponding libcontainer change was needed to make sure
mem-swappiness got reflected in cgroup config
https://github.com/docker/libcontainer/pull/622